spec/models/admin_user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe AdminUser, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
Do what the auto-generated comment says here, and either create a test, or delete this file.